### PR TITLE
Increase test frequency and obtain DR level0 test pass rate

### PR DESCRIPTION
**Pull request by wangke19**
This PR change is caused by pr https://github.com/openshift/release/pull/52752, we want to add DR level0 tests to automated release jobs, before that, we should collect DR cases pass rate(90% we expected) to confirm its stability。

### CODE REVIEW

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-azure-ipi-private-fips-f1-disasterrecovery

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml [hunk 1]
```diff
@@ -647,13 +647,15 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-winc
-- as: aws-ipi-private-fips-f28-disasterrecovery
-  cron: 11 11 20 * *
+- as: aws-ipi-private-fips-f1-disasterrecovery
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:650]: I suggest `f4` to watch it for a week to check its pass ratio, `f1` is a bit terrible for me.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml [hunk 2]
```diff
@@ -703,13 +703,14 @@ tests:
...
+  cron: 11 11 4,8,12,16,20,24,28 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       FIPS_ENABLED: "true"
+      TEST_IMPORTANCE: LEVEL0
     test:
     - ref: idp-htpasswd
     - chain: openshift-e2e-test-qe-disasterrecovery-sanity
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:716]: Do we really want to run the other tests in this chain with frequency f4, or we only want to run LEVEL0 test cases from ginkgo ?

Comment (wangke19) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:716]: We just want to get a results of LEVEL0 test cases for DR, this change is only temporary to get a test pass rate, and if we get the expected results, we will revert to the previous configuration.

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-azure-ipi-private-fips-f4-disasterrecovery

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml (hunk 1 above)
Comment (wangke19) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml]: Updated.

**Comment by wangke19** [comment]
/pj-rehearse  periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-gcp-ipi-disc-priv-fips-f4-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-private-fips-f4-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-gcp-ipi-disc-priv-fips-f4-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-private-fips-f4-disasterrecovery

**Pull request merged**